src/config.json

```
@@ -739,7 +739,9 @@
     "cactus.chat",
     "nfinity.space",
     "catctus.io",
-    "caucus.so"
+    "caucus.so",
+    "forte.io",
+    "rally.io"    
```

These two sites aren't blocked. The whitelist is meant to unblock legitimate sites that end up blocked because of our fuzzy list.

```suggestion
    "caucus.so"
```